circle.yml

```diff
@@ -126,4 +124,9 @@ test:
       unicode
       unicode/utf16
       unicode/utf8
+    - >
+      gopherjs test --short --minify --run='^Test'
+      net/http/cookiejar
+      reflect
+    - gopherjs test --short --minify --run='^Example(MakeFunc|StructTag|StructTag.Lookup|TypeOf)'
```

Comment:
This isn't the approach I meant. Is it possible to override and skip examples similar to how we skip failing/unsupported tests?

Look at https://github.com/gopherjs/gopherjs/commit/3303ee8edfe1a368bdc2c500279f2ba56bf7c3ec for example.

There's no `t.Skip` for examples, but just leaving the func body blank would probably work, right?

My goal is to simplify the command to run tests to just `gopherjs test --short --minify ...` instead of adding more special cases. That way, it's easier to run it locally, not just in CI.